### ISSUE TITLE
AddFunc Mobile Detect (3rd party module)

### ISSUE THREAD

**Issue by joerhoney** [serbanghita/Mobile-Detect#565]
[AddFunc Mobile Detect](https://wordpress.org/plugins/addfunc-mobile-detect/) for WordPress redirects mobile traffic to your mobile website:
- Redirects on a page-by-page basis (posts and custom post types included). For example; yourwebsite.com/contact/ will redirect mobile devices to yourmobilewebsite.com/contact/ instead of always redirecting to your mobile home page.
- This can be overridden on any page individually with a convenient meta box adjacent to the WYSIWYG (usually underneath, by default).
- Sets a cookie to remember which version of your website (desktop or mobile, usually) your visitors opted for (so it's okay to set a link on your mobile site to view the desktop version).
- Provides options to NOT redirect basic parts of WordPress: the Front page (static), Pages, the "Home" page (blog posts page), Posts, Attachments, Authors, Categories, Dates, Tags and search results.
- Includes a widget for inserting a link back to your mobile site, which is only generated for mobile devices.
- Includes two shortcodes for generating links to your mobile site. One is generated only for mobile devices and the other is generated regardless.
- No CSS rules are used and classes are provided, yielding coders full reign to style generated links to fit the website theme. Shortcodes even allow you to set the link's classes.
- Adds a class to the <body> (mobile-detected) to help coders write styles specifically for mobile devices.
- Leaves 404 errors untouched, allowing you to maintain 404 statuses.

Basically, it gives you loads of control over your mobile redirects.
